examples/gallery/lines/hlines_vlines.py

```diff
@@ -0,0 +1,121 @@
+"""
+Horizontal and vertical lines
+=============================
+
+The :meth:`pygmt.Figure.hlines` and :meth:`pygmt.Figure.vlines` methods allow to plot
+horizontal and vertical lines in Cartesian, geographic and polar projections.
+"""
+
+# %%
+# Cartesian coordinate system
+# ---------------------------
+# In Cartesian projections lines are plotted as straight lines.
```

Comment:
Should we say Cartesian/polar/geographic projections or coordinate systems? GMT usually call them "Cartesian/polar coordinate transformations and geographic projections, but I feel Cartesian/polar/geographic coordinate systems may be better?